layouts/shortcodes/open-source.html

```diff
@@ -0,0 +1,14 @@
+<div class="container">
+
+  <h2>Open Source</h2>
+  <p>RAPIDS would not be possible without the collaboration of these important open source projects. Click on a logo to learn more: </p>
+  <div class="is-flex is-flex-wrap-wrap is-justify-content-space-between mt-5">
+      <a href="https://arrow.apache.org/" target="_blank" class="img-w100" > <img src="/images/apache-arrow_Color.png" alt="Apache Arrow"></a>
+      <a href="https://dask.org/" target="_blank" class="img-w100"> <img src="/images/dask_logo.png" alt="Dask"> </a>
+      <a href="https://nuclio.io" target="_blank" class="img-w100"> <img src="/images/nuclio-logo.png" alt="nuclio"> </a>
+      <a href="https://numba.pydata.org/" target="_blank" class="img-w100"> <img src="/images/numba_logo.png" alt="Numba"> </a>
+      <a href="https://scikit-learn.org/stable/index.html" target="_blank" class="img-w100"> <img src="/images/scikit-learn_Color.png" alt="scikitlearn"> </a>
```

Comment:
Why do we list scikit-learn here? Do we have a closer collaboration with sklearn than pandas or networkx, for instance?

Reply:
Its a hold over - we probably should add pandas + networkx?

Reply:
Well, it could also extend to geopandas and scikit-image and scipy (signal). I don’t know what’s right here. Maybe we leave it as-is or remove scikit-learn. cc: @beckernick if you have context here.